# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mepo/0002-zig-0.13-require-zig-0.13-now.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From aa43f762540d25acbdfbd218e5efde4c20851ce1 Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Sat, 8 Jun 2024 02:39:04 +0200
Subject: [PATCH] zig 0.13: require zig 0.13 now

---
 src/main.zig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main.zig b/src/main.zig
index b477ad7..6ba77e2 100644
--- a/src/main.zig
+++ b/src/main.zig
@@ -12,8 +12,8 @@ const utildbg = @import("./util/utildbg.zig");
 pub fn main() !void {
     comptime {
         const v = builtin.zig_version;
-        if (v.major != 0 or v.minor != 12)
-            @panic("Must be built against Zig 0.12.x");
+        if (v.major != 0 or v.minor != 13)
+            @panic("Must be built against Zig 0.13.x");
     }
 
     const allocator = std.heap.c_allocator;
